modDesc.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8" standalone="no" ?>
 <modDesc descVersion="39">
-	<version>5.03.00043</version>
+	<version>5.03.00044</version>
```

Comment:
Sorry I made some pushes version 46 would be next at the time of this writing. Just waiting on a second opinion about this. Thanks for your patience